roles/vdm/tasks/postgres/postgres-instance.yaml

```diff
@@ -143,7 +143,7 @@
     - uninstall
     - update
 
-- name: postgres instance - external post 2022.10
+- name: postgres instance - external post 2022.10 - external cds 2022.12
```

Comment:
@thpang, how's this updated task name?

Reply:
Maybe something like: `external cds post 2022.12`. Just that small change and it'll be good to go